skia-bindings/build_support/binary_cache/utils.rs

```diff
@@ -1,9 +1,25 @@
-use std::io;
-use std::io::Read;
+use std::{
+    fs,
+    io::{self, Error, ErrorKind, Read},
+    path::PathBuf,
+};
 
 /// Download a file from the given URL and return the data.
 pub fn download(url: impl AsRef<str>) -> io::Result<Vec<u8>> {
-    let resp = ureq::get(url.as_ref()).call();
+    let url = url.as_ref();
+
+    // `file` URL, empty hostname, absolute path
+    if let Some(file_path) = url.strip_prefix("file:///") {
```

Comment:
I think there is a compatibility problem on windows platform, since the absolute path on windows is something like `file://C:\path\to\file`. Requiring a file url starts with `file:///` then add extra `/` on the start of the url will change this example path to `/C:\path\to\file`. Obviously, it will break the path.
It is suggested to replace `url.strip_prefix("file:///")` with `url.strip_prefix("file://")` and replace `format!("/{}", file_path)`with `file_path`. Correct me if I'm misunderstood.

Reply:
Yes, that sounds about right.